### ISSUE TITLE
Migrate COUNTER list of robots and crawlers to a COUNTER repository

### ISSUE THREAD

**Issue by COUNTERMetrics** [Project-Counter/cop5#251]
We will migrate the Bots list from its current location to the COUNTER GitHub so that we have control over it, and update the list to encompass a more comprehensive set of bots.

**Comment by COUNTERMetrics** [comment]
The AI working group reviewed the question of bot definition and agreed that the bots list should move to a new GitHub repo owned by COUNTER.

The plan is for the new [https://github.com/Project-Counter/counter-bots](https://github.com/Project-Counter/counter-bots) to draw on [https://github.com/ai-robots-txt/](https://github.com/ai-robots-txt/). As part of the next release, we need to (a) populate the list, and (b) ensure Section 7.8 points to the new location.

The information will be available in .txt and .json format. Report providers and consumers will be able to request additions to the list using the Issues functionality. 

Outstanding query for Code team about whether updates to the list will need to be tied to the usual fix release schedule.

**Comment by berndoberknapp** [comment]
The ai-robots list does not include non-AI bots like the usual Googlebot. Will this list be merged with the current COUNTER robots list or will the non-AI bots be addressed in some other way?

**Comment by COUNTERMetrics** [comment]
Yes, the intention is to incorporate the existing bot list plus other bots that have been flagged to COUNTER.

**Comment by dave-siq** [comment]
Have we created a repo for this?

**Comment by berndoberknapp** [comment]
See the comment from Nov 14, 2025 above. This is probably the most complex PRs we have for R5.1.1, it will require rewriting Section 7.8 and Appendix H and of course setting up the bots list in the new repository. Therefore I've assigned the issue to @COUNTERMetrics.

**Comment by COUNTERMetrics** [comment]
I think we can [mirror](https://github.com/marketplace/actions/mirroring-repository) [ai-robots-txt](https://github.com/ai-robots-txt/ai.robots.txt), which might simplify matters for us in future. @berndoberknapp what do you think?
